### PR TITLE
feat(KTabs, KTextArea, KToaster): add types [KHCP-6974]

### PR DESCRIPTION
**Pull request by DariaYeremina**
# Summary

https://konghq.atlassian.net/browse/KHCP-6974
Add types for KTabs, KTextArea, KToaster

## PR Checklist

* [ ] Does not introduce dependencies
* [ ] **Functional:** all changes do not break existing APIs and if so, bump major version.
* [ ] **Tests pass:** check the output of yarn test
* [ ] **Naming:** the files and the method and prop variables use the same naming conventions as other Kongponents
* [ ] **Framework style:** abides by the essential rules in Vue's style guide
* [ ] **Cleanliness:** does not have formatting issues, unused code (e.g., console.logs, debugger), or leftover comments
* [ ] **Docs:** includes a technically accurate README, uses JSDOC where appropriate

### CODE REVIEW

File: src/components/KToaster/ToastManager.ts [hunk 1]
```diff
@@ -43,18 +43,18 @@ export default class ToastManager {
     createApp(Toast).mount(`#${this.id}`)
   }
 
-  setTimer(key: any, timeout: number) {
+  setTimer(key: any, timeout: number): number {
```
Comment (adamdehaven) [inline src/components/KToaster/ToastManager.ts:46]: question: why is the return type `number` here?

File: src/types/text-area.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+export interface TextAreaLimitExceed {
+  value: string,
+  length: number,
+  characterLimit: number,
+  limitExceeded: boolean,
```
Comment (adamdehaven) [inline src/types/text-area.ts:5]: nit: no commas

File: src/components/KToaster/ToastManager.ts (hunk 1 above)
Comment (DariaYeremina) [reply src/components/KToaster/ToastManager.ts:46]: https://developer.mozilla.org/en-US/docs/Web/API/setTimeout#return_value

File: src/types/text-area.ts (hunk 2 above)
Comment (DariaYeremina) [reply src/types/text-area.ts]: fixed

**Review by kaiarrowood — APPROVED** [review]
LGTM! 🚀

**Pull request merged**